websites/B/BoxSeries/presence.ts

```diff
@@ -0,0 +1,162 @@
+const presence = new Presence({
+		clientId: "955350055034945576"
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+		browsing: "general.browsing"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video = {
+	current: 0,
+	duration: 0,
+	paused: true
+};
+
+presence.on(
+	"iFrameData",
+	(data: { current: number; duration: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const [logo, cover, privacy, buttons, timestamps] = await Promise.all([
+			presence.getSetting<number>("logo"),
+			presence.getSetting<boolean>("cover"),
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("timestamps")
+		]),
+		titleMain =
+			document.querySelector("#dt_contenedor > div.module > div > div > h1")
+				?.textContent ?? "pesquisa não encontrada",
+		titleEp =
+			document.querySelector("#single > h1.epih1")?.textContent.split(" - ") ??
+			"pesquisa não encontrada",
+		pathArray = document.location.toString().split("/"),
+		presenceData: PresenceData = {
+			largeImageKey: ["default", "default_horizontal_tp"][logo],
+			startTimestamp: browsingTimestamp
+		};
+	if (location.search.includes("s")) {
```

Comment:
probs using [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) is better